ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -185,14 +184,18 @@ Get Model Route via UI
 Open ${section} Options Menu
     [Documentation]    Opens the "Options menu" dropdown for different sections
     ...    Valid sections are: "Name", "Model framework (name - version)", "Project"
-    Click Element    xpath://span[.="${section}"]/../../..//button  # Remove attribute which is not present anymore
+    ${optionsmenu}=    Run Keyword And Return Status    Page Should Contain Element
+    ...    xpath://span[.="${section}"]/../../..//button[@aria-label="Options menu"]
+    IF    ${optionsmenu} == ${TRUE}
+        SeleniumLibrary.Click Element    xpath://span[.="${section}"]/../../..//button[@aria-label="Options menu"]  # Remove attribute which is not present anymore
```

Comment:
This aria-label attribute was removed in 1.34 RC, why are you putting it back in?